Authentication.md

```diff
@@ -304,6 +305,150 @@ if err != nil {
+2. Second, the authenticator uses the grant type `urn:ibm:params:oauth:grant-type:assume` to obtain a bearer token
+that reflects the identity of the trusted profile, passing in the initial bearer token
+from the first step, along with the trusted profile-related inputs.
+
+The authenticator will also obtain a new bearer token when the current token expires.
+The bearer token is then added to each outbound request in the `Authorization` header in the
+form:
+```
+   Authorization: Bearer <bearer-token>
+```
+
+### Properties
+
+- ApiKey: (required) the IAM apikey to be used to obtain the initial IAM access token.
+
+- IAMProfileCRN: (optional) the Cloud Resource Name (CRN) associated with the trusted profile
+for which an access token should be fetched.
+Exactly one of IAMProfileCRN, IAMProfileID or IAMProfileName must be specified.
+
+- IAMProfileID: (optional) the ID associated with the trusted profile
+for which an access token should be fetched.
+Exactly one of IAMProfileCRN, IAMProfileID or IAMProfileName must be specified.
+
+- IAMProfileName: (optional) the name associated with the trusted profile
+for which an access token should be fetched.  When specifying this property, you must also
+specify the IAMAccountID property as well.
+Exactly one of IAMProfileCRN, IAMProfileID or IAMProfileName must be specified.
+
+- IAMAccountID: (optional) the ID associated with the IAM account that contains the trusted profile
+referenced by the IAMProfileName property.  The IAMAccountID property must be specified if and only if
+the IAMProfileName property is specified.
+
+- URL: (optional) The base endpoint URL of the IAM token service.
+The default value of this property is the "prod" IAM token service endpoint
+(`https://iam.cloud.ibm.com`).
+Make sure that you use an IAM token service endpoint that is appropriate for the
+location of the service being used by your application.
+For example, if you are using an instance of a service in the "production" environment
+(e.g. `https://resource-controller.cloud.ibm.com`),
+then the default "prod" IAM token service endpoint should suffice.
+However, if your application is using an instance of a service in the "staging" environment
+(e.g. `https://resource-controller.test.cloud.ibm.com`),
+then you would also need to configure the authenticator to use the IAM token service "staging"
+endpoint as well (`https://iam.test.cloud.ibm.com`).
+
+- ClientId/ClientSecret: (optional) The `ClientId` and `ClientSecret` fields are used to form a 
+"basic auth" Authorization header for interactions with the IAM token server when fetching the
+initial IAM access token. These fields are optional, but must be specified together.
+
+- Scope: (optional) the scope to be used when obtaining the initial IAM access token.
+If not specified, then no scope will be associated with the access token.
+
+- DisableSSLVerification: (optional) A flag that indicates whether verificaton of the server's SSL 
```

Comment:
I fixed this one, plus the four already-existing instances of the same typo :joy:

Reply:
Ohh, I didn't check the the other files. Good catch, haha!

Reply:
Those other instances already existed in Authentication.md prior to my recent changes.  Ya know... it's a good thing (sort of) that I basically just copy/pasted the Iam-related content when developing the IamAssume doc content for this PR, because we might not have caught those other typos otherwise :joy: